packages/dashboard/src/customization/widgets/lineScatterChart/component.tsx

```diff
@@ -151,6 +154,22 @@ const LineScatterChartWidgetComponent: React.FC<LineScatterChartWidget> = (widge
   // the 44 is from the widget tile header's height
   const size = { width: chartSize.width, height: chartSize.height - 44 };
 
+  const onChartOptionsChange = (options: Omit<ChartOptions, 'queries'>) => {
```

Comment:
ChartOptions has a lot of fields... why do we need all those ? can this type of just legend or sizeOption ? given all you need is the height and width ?